src/screens/Naver/NaverDetails.tsx

```diff
@@ -16,19 +16,52 @@ const NaverDetails: FC = ({ route }) => {
   const [messageModal, setMessageModal] = useState<string>('');
   const navigation = useNavigation();
 
+  const getJobTime = admission_year => {
+    const diffInMillisecond = new Date().valueOf() - admission_year.valueOf();
+
+    const year_age = Math.floor(diffInMillisecond / 31536000000);
+    const day_age = Math.floor((diffInMillisecond % 31536000000) / 86400000);
+    const month_age = Math.floor(day_age / 30);
+
+    let date = '';
+
+    if (day_age === 0) {
+      return 'Contratado hoje';
+    }
+
+    if (year_age > 0) {
+      date = year_age >= 1 ? '' + year_age + ' ano - ' : '' + year_age + ' anos - ';
+    }
+
+    if (month_age > 0) {
+      date = date + (month_age === 1 ? month_age + ' mes - ' : month_age + ' mesês - ');
+    }
+
+    if (day_age > 0) {
```

Comment:
Duas dicas, pra concatenar você poderia ter utilizado o +=.
Ou então, ter utilizado template string ` pra fazer as validações dos dados dentro da string em si.

Reply:
opa, to fazendo essas alterações